<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityReceipt</class>
 <widget class="QMainWindow" name="ActivityReceipt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="labelCostAll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Blokk végösszege:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CurrencySpinBox" name="spinBoxCostAll">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 216, 230);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="specialValueText">
         <string/>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelReceiptDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Blokk dátuma:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 216, 230);</string>
        </property>
        <property name="locale">
         <locale language="Hungarian" country="Hungary"/>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy.MMMM.dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="timeSpec">
         <enum>Qt::LocalTime</enum>
        </property>
        <property name="date">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Blokk felbontása</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelCostItem">
           <property name="styleSheet">
            <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
</string>
           </property>
           <property name="text">
            <string>Tétel összege:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CurrencySpinBox" name="spinBoxCosItem">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(173, 216, 230);</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="specialValueText">
            <string/>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelCategory">
           <property name="styleSheet">
            <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Kategória:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxCategories">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(173, 216, 230);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelComment">
           <property name="styleSheet">
            <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Leírás:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditComment">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(173, 216, 230);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAddItem">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(173, 216, 230);
font: 700 9pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Tétel hozzáadása</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableView" name="tableView">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRemoveItem">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(173, 216, 230);
font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Elem törlése</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRemoveAll">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(173, 216, 230);
font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Minden elem törlése</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSave">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(173, 216, 230);
font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Mentés</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CurrencySpinBox</class>
   <extends>QSpinBox</extends>
   <header>c:/Users/Dani/Qt-source/FamilyFinancies/FamilyFinancies/currencyspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
